integration/shared/isolated/add_network_policy_command_test.go

```diff
@@ -119,15 +112,12 @@ var _ = Describe("add-network-policy command", func() {
 					Eventually(session).Should(Say("OK"))
 					Eventually(session).Should(Exit(0))
 
-					session = helpers.CF("curl", fmt.Sprintf("/networking/v1/external/policies?id=%s", sourceAppGUID))
+					session = helpers.CF("network-policies")
+					Eventually(session).Should(Say(`Listing network policies in org %s / space %s as %s\.\.\.`, sourceOrg, sourceSpace, username))
+					Consistently(session).ShouldNot(Say("OK"))
```

Comment:
I'm not sure I understand what this assertion is for. It doesn't seem like there's ever a scenario where this command will print `OK`, so having a standalone negative assertion feels strange to me.

Reply:
Agreed, will remove.